### PR TITLE
feat(component): new file-input component

### PR DESCRIPTION
**Pull request by vasucp1207**
- File upload example
- Helper text
- Sizes
- Dropone

### CODE REVIEW

**Comment by cogor** [comment]
May you add an empty line at end of file?
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

**Comment by cogor** [comment]
Drop zone doesn't work, can we fix it?
I found an article on [MDN](https://developer.mozilla.org/en-US/docs/Web/API/HTML_Drag_and_Drop_API/File_drag_and_drop) and think it will be heplful

**Comment by vasucp1207** [comment]
It's actually working but the value does not appear in dropzone, I fix this now it's working fine.

**Comment by cogor** [comment]
Incorrect title in docs – "Vue Dropdown - Flowbite "

**Comment by cogor** [comment]
And drag and drop files isn't works yet
[Look this video](https://imgur.com/a/ociuYVS), when I droped file it just opens in tab

**Comment by vasucp1207** [comment]
> And drag and drop files isn't works yet [Look this video](https://imgur.com/a/ociuYVS), when I droped file it just opens in tab

Fix the file drag part according to the MDN check now.

**Review by cogor — CHANGES_REQUESTED** [review]

**Comment by cogor** [comment]
I found some problems: 
Instead of file I got filename and I can't do something with it, for example upload to server
We need to add multiple files support

File: src/components/FileInput/FileInput.vue [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  },
+})
+
+const handleChange = (event: Event) => {
+  const target = event.target as HTMLInputElement
+  model.value = target.files?.[0]?.name
+}
+
+const dropFileHandler = (event: any) => {
+  console.log('File(s) dropped')
```
Comment (cogor) [inline src/components/FileInput/FileInput.vue:66]:
```suggestion
```

**Comment by vasucp1207** [comment]
> I found some problems: Instead of file I got filename and I can't do something with it, for example upload to server We need to add multiple files support

Add support for multiple files, also changes file type to actual **file** object, not fileName.

**Review by cogor — APPROVED** [review]

**Comment by cogor** [comment]
LGTM

**Pull request merged**